packages/jsActions/nanoflow-actions-native/src/geolocation/GetCurrentLocation.ts

```diff
@@ -5,11 +5,14 @@
 // Other code you write will be lost the next time you deploy the project.
 
 import { Big } from "big.js";
+import { Platform, NativeModules } from "react-native";
 import Geolocation, {
     GeolocationError,
     GeolocationOptions,
-    GeolocationResponse
+    GeolocationResponse,
+    GeolocationStatic
 } from "@react-native-community/geolocation";
+import type { GeolocationServiceStatic, GeoError, GeoPosition, GeoOptions } from "../../typings/Geolocation";
```

Comment:
I see you used the type syntax but kept the typings file - with the type keyword, you can import from the dependency that exports these types and remove your user-defined types file. Does this work for you?